pkg/app/app.go

```diff
@@ -118,6 +119,18 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	if len(bundle) > 0 {
 		inputFormat = "bundle"
 		opt.InputFiles = []string{bundle}
+	} else if len(opt.InputFiles) == 0 {
+		// Here docker-compose is the input
+		opt.InputFiles = []string{"docker-compose.yml"}
+		_, err := os.Stat("docker-compose.yml")
+		if err != nil {
+			logrus.Debugf("'docker-compose.yml' not found: %v", err)
+			opt.InputFiles = []string{"docker-compose.yaml"}
+			_, err = os.Stat("docker-compose.yaml")
+			if err != nil {
+				logrus.Fatalf("No 'docker-compose' file found: %v", err)
+			}
+		}
```

Comment:
This above code is copy-and-pasted from the `ValidateUpDown` function. I think this could be refactored a bit so you can use the function in both places.